packages/admin/dashboard/src/routes/tax-regions/tax-region-tax-override-create/components/tax-region-override-create-form/tax-region-tax-override-create.tsx

```diff
@@ -389,7 +389,8 @@ export const TaxRegionCreateTaxOverrideForm = ({
                             <Form.Control>
                               <PercentageInput
                                 {...field}
-                                placeholder="0.00"
+                                placeholder="0.0000"
```

Comment:
todo: let's not change the placeholder; four decimals is pretty unusual for a tax rate, so no need to have that as the placeholder